package.json

```diff
@@ -52,7 +52,8 @@
     "test-watch": "npm run test-compile -- -watch",
     "test-compile": "tsc -p ./",
     "pretest": "npm run test-compile",
-    "test": "node ./out/test/runTest.js"
+    "test": "node ./out/test/runTest.js",
+    "deploy": "vsce publish --yarn"
```

Comment:
Would we put in a place holder for the version number

Reply:
version number is based on the package.json

Reply:
Sounds cool :+1: